pages/data-providers/[data-provider-id]/deposit-dates.jsx

```diff
@@ -91,6 +90,41 @@ const DepositDates = ({
         )}
     </Card>
 
+    <Card tag="section">
+      <h2>Publication dates</h2>
+      <p>Compares publications dates with Crossref</p>
+      {store.depositDates.publicationDatesValidate ? (
+        <StackedVerticalBarChart
+          data={[
+            {
+              value: store.depositDates.publicationDatesValidate.fullCount,
+              caption: 'Matching',
```

Comment:
Do you mind moving these messages to `texts`? It can be done later separately if you wish.

Reply:
I will do it later together with Card.Description

Reply:
Oh, it was already merged. Gonna do it here...